app/models/account.rb

```diff
@@ -0,0 +1,3 @@
+class Account < ApplicationRecord
+    belongs_to :user
```

Comment:
validations can go here

```ruby
validates :name, presence: true, uniqueness: true
```

Reply:
[Check out Active Record Validations](http://guides.rubyonrails.org/active_record_validations.html)

Reply:
Also, add a `presence` validation for `:balance`